include/CppUTest/UtestMacros.h

```diff
@@ -232,6 +232,18 @@
 #define BYTES_EQUAL_TEXT(expected, actual, text)\
     LONGS_EQUAL_TEXT((expected) & 0xff, (actual) & 0xff, text)
```

Comment:
This one will need to be changed to one of the others. But you want to do that later, right?